src/rgw/rgw_crypt.cc

```diff
@@ -1161,6 +1161,7 @@ int rgw_s3_prepare_encrypt(struct req_state* s,
 	}
       } catch (const buffer::error& e) {
         ldpp_dout(s, 5) << __func__ <<  "decode bucket_encryption_conf failed" << dendl;
```

Comment:
Should we put this dump to ldpp_dout(this, 0) as it is an error scenario?

Reply:
I have used the same number as used at other places in the same function. I think it is right. For example, 
std::string_view key_id =
        get_crypt_attribute(s->info.env, parts,
                            X_AMZ_SERVER_SIDE_ENCRYPTION_AWS_KMS_KEY_ID);
      if (!key_id.empty()) {
        ldpp_dout(s, 5) << "ERROR: SSE-KMS encryption request is missing the header "
                         << "x-amz-server-side-encryption"
                         << dendl;
        s->err.message = "Server Side Encryption with KMS managed key requires "
                         "HTTP header x-amz-server-side-encryption : aws:kms";
        return -EINVAL;
      }

Reply:
No, in other places in the same function they have used ldpp_dout(this, 5).

Reply:
Ok. I think they are not consistent with this. I found it to be (this, 0) so I used that. Resolving this.